tools/autograd/load_derivatives.py

```diff
@@ -137,7 +137,7 @@ def postprocess_forward_derivatives(
     def find_required_inputs(formula: str, postfix: str) -> Tuple[str, ...]:
         required_inputs = set()
         for arg in args_with_derivatives:
-            if arg.type == 'TensorList':
+            if arg.type == 'at::TensorList':
```

Comment:
Is this a bugfix? (Should this be `arg.type == 'at::TensorList' or arg.type == 'TensorList'`?)

Reply:
This is a bugfix indeed. This code wasn't actually used before and it is now that we have the "cat" formula.

Reply:
Should we `assert 'TensorList' not in arg.type` after this if statement? Would make it easier to catch bugs like this in the future

Reply:
I don't think so. It was just me not knowing that the namespace is included in this name.
I think there should be methods on the type to check that directly but I couldn't find it.